lib/cn/csched_sp3.c

```diff
@@ -5,6 +5,7 @@
 
 #define MTF_MOCK_IMPL_csched_sp3
 
+#include <math.h>
```

Comment:
tgmath.h for the win!

Reply:
Oh nice, forgot about that, think I've used it only once!

Reply:
It is new to me in the past year. Really makes dealing with the C math function a lot easier.